gomod-sync/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
```

Comment:
I think it would be good to include vendor in the gitignore just in case someone runs in vendored mode.

Reply:
Sounds good!

Reply:
Comment removed.